pshtt.py

```diff
@@ -149,13 +150,17 @@ def basic_check(endpoint):
         except requests.exceptions.SSLError:
             # If it's a protocol error or other, it's not live.
             endpoint.live = False
+            logging.warn("Unexpected SSL protocol (or other) error during retry.")
             return
         except requests.exceptions.RequestException:
             endpoint.live = False
-            logging.warn("Unexpected requests exception during retry.")
+            logging.warn("Unexpected requests exception during retry. Printing error:")
+            logging.warn(utils.format_last_exception())
```

Comment:
Now displaying generic TLS connection errors to console during the run, even without `--debug` or `-d`.